learning_resources_search/constants.py

```diff
@@ -111,6 +111,7 @@ class FilterConfig:
         },
     },
     "free": {"type": "boolean"},
+    "is_learning_material": {"type": "boolean"},
```

Comment:
is_learning_material is staying in the index (but not shown in the results) since it is used in the default search. I forgot to add it to the mapping previously